src/com/owncloud/android/ui/fragment/ExtendedListFragment.java

```diff
@@ -64,6 +69,11 @@
     private SwipeRefreshLayout mRefreshGridLayout;
     private SwipeRefreshLayout mRefreshEmptyLayout;
     private TextView mEmptyListMessage;
+
+    private FloatingActionsMenu fabMain;
```

Comment:
Please, let's use the ```m``` prefix for member variables. That helps to recognize members in code snippets in Github, where syntax highlight is not applied to them.